butane_codegen/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "butane_codegen"
-version = "0.5.0"
+version = "0.5.1-alpha.1"
```

Comment:
```suggestion
version = "0.5.2-alpha.1"
```

Reply:
https://crates.io/crates/butane_codegen is a patch release behind the others